### PR TITLE
[frameworks][examples] Add Elm (create-elm-app)

### PR DESCRIPTION
**Pull request by connorads**
This adds [Elm](http://elm-lang.org/) ([create-elm-app](https://github.com/halfzebra/create-elm-app)) to the zero-config frameworks.

Ran `now-static-build` integration test for `61-create-elm-app`:
https://test2020-cho4fha02.vercel.app/
image

Related to https://github.com/vercel/docs/issues/1530

### CODE REVIEW

File: packages/frameworks/frameworks.json [hunk 1]
```diff
@@ -797,6 +797,34 @@
...
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"create-elm-app\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`npm run build` or `elm-app build`"
+      },
+      "devCommand": {
+        "value": "elm-app start"
```
Comment (connorads) [inline packages/frameworks/frameworks.json:853]: What about `PORT`?

I don't believe there's a way to pass it via command line and `create-elm-app` reads the `PORT` environment variable implicitly.

https://github.com/halfzebra/create-elm-app/blob/f09a3f7183712646eb5ba0a983136b5283d3656c/scripts/start.js#L150

Comment (styfle) [reply packages/frameworks/frameworks.json:853]: If the elm cli can detect the `PORT` environment variable, then this will work as-is 👍

**Comment by mcsdevv** [comment]
> Hey @mcsdevv, is there anything I can do to help get this reviewed and/or merged?

Hey there! We'll do our best to get back to you very shortly on this.

**Comment by mcsdevv** [comment]
Hey @connorads just so you know this is on our radar to get done but being tracked internally. We'll let you know as soon as we've made some progress.

**Comment by connorads** [comment]
I tried merging `master` and working on this locally for a few hours but couldn't get it to work (I tried putting it in `packages/now-node` 😅🤷‍♂️)
And then I found #5640
Is this dead now @mcsdevv @styfle? 🤔

**Comment by alissonfpmorais** [comment]
Hey guys, hope you all are good!
Is there any progress on this issue? If there's something I could help, I'll be glad to do :)

**Comment by styfle** [comment]
There have been some changes since this PR was created.

You no longer need to manually add frameworks to Vercel, instead you can update the framework to output `.vercel_build_output`.

Learn More: https://vercel.com/docs/more/adding-your-framework

**Pull request closed without merging**